storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -227,6 +231,57 @@ public static boolean showSkipOnCompactNotification() {
         return showButtonOnCompactNotification(NOTIFICATION_BUTTON_SKIP);
     }
 
+    public static boolean showNextChapterOnCompactNotification() {
+        return showButtonOnCompactNotification(NOTIFICATION_BUTTON_SKIP);
```

Comment:
The preferences for the "Next Chapter" button are still using the `NOTIFICATION_BUTTON_SKIP` integer, not `NOTIFICATION_BUTTON_NEXT_CHAPTER`, which is what I'd expect. I think this is why the next chapter button is re-appearing when not set.

Reply:
:man_facepalming: thanks for noticing this.